.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,128 @@
+# 0010 - Interest Bearing Channel Application
+
+# Status
+
+Accepted
+
+# Context
+
+For a permissionless state channel network (SCN) to be sustainable, incentive structures must exist for each class of system actor (service providers, service consumers, & liquidity providers).
+
+Our work generally presumes the existence of incentives that motivate service providers & consumers. The design work for us is in mechanisms to incentivize intermediaries (liquidity providers) into the network.
+
+# Prior Art
+
+The Bitcoin Lightning Network (LN) is an in-production payment channel network (PCN) with a fee mechanism.
```

Comment:
Is it worth having a "definitions" section for LN, PCN, SCN, ... ? Don't feel strongly.

Reply:
Going make assumptions about the audience that say the given context should suffice.